rfcs/experimental/dynamic_selection/README.md

```diff
@@ -0,0 +1,516 @@
+| `p.submit(f, args…)` | Returns `submission_t<T>` that satisfies [Submission](#submission_req_id). The function invokes `f` but does not wait for the `wait_t<T>` it returns to complete. |
+| `p.submit_and_wait(f, args…)` | Returns `void`. The function invokes `f` and waits for the `wait_t<T>` it returns to complete. |
+
+<a id="selection_req_id"></a>
+### Selection
+
+The type `T` satisfies *Selection* for a given [Policy](#policy_req_id) `p` if given,
+
+- `s` an arbitrary identifier of type `T`
+- `i` an object of type `Info` where `execution_info_v<Info>` is `true`
+- 'v' an object of type `V` and `is_same_v<Info::value_type, V>` is `true`
+
+| *Must* be well-formed | Description |
+| --------------------- | ----------- |
+| `s.unwrap()` | Returns `resource_t<T>` that should represent one of the resources returned by `p.get_resources()` for the Policy `p` that generated `s`. |
+| `s.policy()` | Returns the Policy `p` that was used to make the selection. |
+| `s.report(i)` | Returns `void`. Notifies policy that an execution info event has occurred. |
+| `report_execution_info<T, Info>::value`, `report_execution_info_v<T,Info>` | `true` if this selection needs the backend to report the Info. `false` otherwise. |
+
+<a id="submission_req_id"></a>
+### Submission
+
+The type `T` satisfies *Submission* for a given [Policy](#policy_req_id) `p` if given,
+
+- `s` an arbitrary identifier of type `T`
+
+| *Must* be well-formed | Description |
+| --------------------- | ----------- |
+| `s.get_policy()` | Returns an object that satisfies [Policy](#policy_req_id) and corresponds to the Policy that made the selection. |
+| `s.wait()` | Blocks until the submission has completed. |
+| `s.unwrap()` | Returns the underlying backend type value. This type may be void, may represent the backend’s synchronization type, or may represent a return value from the submission. |
+
+## Provided Concrete Policies 
+
+<a id="fixed_resource_id"></a>
+### `fixed_resource_policy`
+
+```cpp
+template<typename Backend=sycl_backend> 
+class fixed_resource_policy;
+```
+
+| Constructors and Initialization |
+| -----------------------------------------------------------|
+| `fixed_resource_policy(deferred_initialization_t); // (1)` |
+| `fixed_resource_policy(size_t index=0); // (2)` |
+| `fixed_resource_policy(const std::vector<resource_t<Backend>>& resources, size_t index=0); // (3)` |
+| `void initialize(size_t index=0); // (4)` |
+| `void initialize(const std::vector<resource_t<Backend>>& resources, size_t index=0); // (5)` |
+
+1. Defers initialization and requires a later call to `initialize`.
+2. Always selects index `index` in the default set of resources.
+3. Uses the provided set of resources and always selects index `index` from that set.
+4. Always selects index `index` in the default set of resources.
+5. Uses the provided set of resources and always selects index `index` from that set.
```

Comment:
The choice of formatting which combines a table and a list seems strange. Was it supposed to be a code block and a list?

Reply:
I agree, code block will work better here. I will change.

Reply:
Section will be simplified.